### ISSUE TITLE
Feature request: button for resetting the panels

### ISSUE THREAD

**Issue by ignoreigor** [tine20/tine20#7204]
After having some fun, because the toolbar for mail operations and quickfilters had disappeared for one user, I could finally solve the problem by deleting some records from tine20_state table (after trying different browsers, disabling browser extensions, clearing cached browser data, clearing server's cached and temporary data and so on). 
Maybe it would be nice to have a button for both users and administrators in the gui that resets the panel settings for him-/herself and specific user respectively.  I don't think that it is neccessary to distinguish or offer this reset function for a single application's panel. Could also be a quickfix for misresized columns - the width adaption is a bit tricky.

**Comment by pschuele** [comment]
yup, sounds like a good idea. we could also add a CLI api for this - which could be implemented very quickly.

**Comment by ignoreigor** [comment]
Hi Philipp, 
thank you for observing and labeling this request. Of course one can realize this through a CLI command, but in this case you can execute a query. For the beginning a CLI command is surely not bad, more safe and comfortable than working with the database. 

This would also be good for the column grids, as time goes by the width of the columns changes, probably due to the different screen widths, and the column's resizing can be quite annoying. Maybe a user can sometime perform the reset for him-/herself.

**Comment by pschuele** [comment]
yep, i agree - maybe I can add a controller function with acl check and the cli if I find a moment. adding it to the client would require some thoughts about placing the button at a convient spot in the grid controls. do you have an idea where it should be?

**Comment by ignoreigor** [comment]
Hi Philipp,

I think in the GUI the function is more useful for the user than for the administrator. I wouldn't distinguish for which application the panel is reset, that's going too far, just do a complete reset of the GUI. If one user* has customized the interface too much, one has to do it differently. If one just want a "clean start", reset and adjust one or two things, if necessary.
So I would offer the button centrally, either in the user's context menu, where one can edit one's profile and change password - or, to avoid accidental triggering, in the profile settings -> General Settings -> Tinebase - if you can put buttons in there.

For the admin, if this has to be done in the GUI, either in the context menu of the user administration (then you could kill several users with it) or somewhere in a panel of the user dialog, since multiple editing is not provided here.

**Comment by pschuele** [comment]
thanks for your input! I hope, we can make some progress on the implementation soon.
